cms/templates/js/highlights-enable-editor.underscore

```diff
@@ -0,0 +1,24 @@
+<p>
+<% // xss-lint: disable=underscore-not-escaped %>
+<%= edx.HtmlUtils.interpolateHtml(
```

Comment:
This doesn't need a call to `interpolateHtml` since there's no interpolation of strings happening here.  You can then also delete the disabling of the warning above.